backends/test/suite/runner.py

```diff
@@ -40,6 +41,16 @@
 }
 
 
+def _get_test_seed(test_base_name: str) -> int:
```

Comment:
Why not set a new, global seed every run? And print it somewhere to reproduce. Hardcoding seed ==> we will test with same random numbers every time, not sure if that's what we want.

Reply:
Update to generate a random run-wide seed and allow specifying a seed from CLI.